katas/metal-slug/mateo/soldier.test.js

```diff
@@ -0,0 +1,29 @@
+import { Soldier } from './soldier';
+import { Vehicle } from './vehicle';
+import { Weapon } from './weapon';
+
+describe('this test is for soldier class', () => {
```

Comment:
Please improve test descriptions trying to avoid technical words (test/class) and use more natural language to describe the behavior being tested instead